### PR TITLE
docs: add Azure Firewall scrape

### PR DESCRIPTION
**Pull request by locmai**
Update for https://github.com/tomkerkhove/promitor/pull/2545

### CODE REVIEW

File: docs/scraping/providers/azure-firewall.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+tags:
+  - Scraper
+  - Resource Discovery
+  - Azure Firewall
+---
+
+# Azure Automation account
```
Comment (tomkerkhove) [inline docs/scraping/providers/azure-firewall.md:8]: Can you align please?

File: docs/scraping/providers/azure-firewall.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.1-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
```
Comment (tomkerkhove) [inline docs/scraping/providers/azure-firewall.md:10]:
```suggestion
![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.12-green.svg)![Resource Discovery Support Badge](https://img.shields.io/badge/Support%20for%20Resource%20Discovery-Yes-green.svg)
```

File: docs/scraping/providers/azure-firewall.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+You can scrape an Azure Automation account via the `AzureFirewall`
```
Comment (tomkerkhove) [inline docs/scraping/providers/azure-firewall.md:12]:
```suggestion
You can scrape an Azure Firewall via the `AzureFirewall`
```

File: docs/scraping/providers/azure-firewall.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+ resource type.
+
+When using declared resources, the following fields need to be provided:
+
+- `azureFirewallName` - The name of the Azure Firewall.
+
+All supported metrics are documented in the official [Azure Monitor documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/reference/supported-metrics/microsoft-network-azurefirewalls-metrics).
+
+
+## Example
```
Comment (tomkerkhove) [inline docs/scraping/providers/azure-firewall.md:22]: Nit
```suggestion
## Example
```

File: docs/scraping/providers/azure-firewall.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+Here is an example configuration:
+
+
```
Comment (tomkerkhove) [inline docs/scraping/providers/azure-firewall.md:26]: Nit
```suggestion

```

**Review by tomkerkhove — CHANGES_REQUESTED** [review]

File: docs/scraping/providers/azure-firewall.md (hunk 1 above)
Comment (locmai) [reply docs/scraping/providers/azure-firewall.md]: Hi, fixed!

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**